cs_upload.py

```diff
@@ -79,6 +140,9 @@ def generate_trace_upload_post(self) -> Tuple[str, str]:
         return (object_storage_url, form_data)
 
     def create_trace(self, project_id: str, name: str, key: str, size: int) -> str:
+        logger.info(
+            f"Registering trace in CAP (project ID: {project_id}, name: {name})"
```

Comment:
How about providing the decoded project ID here, rather than the opaque GraphQL ID? We could pass the integer database ID around and only call `to_global_id()` here while assembling the query.

Reply:
**nitpick, non-breaking**: What you've done certainly works, but if I'm not mistaken, we're reading the integer database IDs off the command-line, converting them to opaque IDs, passing those as arguments to the different functions, and then converting them back to integer database IDs for including in the logs. Wouldn't it have been simpler to work with the integer values and only convert to opaque GraphQL IDs at the point of use, when building and sending a GraphQL query to the API? What do you think?

Reply:
I applied your suggestion.

I think both approaches are valid but we can always change later if one turns out to be better.